doap.rdf

```diff
@@ -58,6 +58,16 @@
     <category rdf:resource="http://projects.apache.org/category/network-client" />
     <category rdf:resource="http://projects.apache.org/category/network-server" />
     <release rdf:parseType="Collection">
+      <Version>
```

Comment:
This change should get back into master.

How are we going to separate out the changes that should get back into master from the changes that should not if they are all in the same commit?

Reply:
Master needs to be templated so we are not hand editing 40 files for a release, this is tedious and error prone and not a good path forward. Next release needs to be automated, I'll bring this topic up on the dev@ list and we can come up with a good way to move it forward